.github/stale.yml

```diff
@@ -3,6 +3,7 @@ daysUntilClose: 7
 exemptLabels:
   - Bug
   - Critical
+  - Unstaleable
```

Comment:
Is this a real word? Is this just fantasy?

What do you think about `Do not close`?

Reply:
It's totally my fantasy 😄 `Do not close` is pretty good for me, says exactly what should (not) be done 👌